features/unsupported/USBHost/USBHostMSD/USBHostMSD.cpp

```diff
@@ -293,20 +307,20 @@ int USBHostMSD::dataTransfer(uint8_t * buf, uint32_t block, uint8_t nbBlock, int
     return SCSITransfer(cmd, 10, direction, buf, blockSize*nbBlock);
 }
 
-int USBHostMSD::getMaxLun() {
+int USBHostMSD::getMaxLun()
+{
     uint8_t buf[1], res;
     res = host->controlRead(    dev, USB_RECIPIENT_INTERFACE | USB_DEVICE_TO_HOST | USB_REQUEST_TYPE_CLASS,
                                 0xfe, 0, msd_intf, buf, 1);
     USB_DBG("max lun: %d", buf[0]);
     return res;
 }
 
-int USBHostMSD::disk_initialize() {
+int USBHostMSD::init()
```

Comment:
I ll take it account in my next PR. (the fact that USB is not in good style does not help ...)